backend/ops_api/.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 select = B,B9,BLK,C,E,F,S,W
-ignore = E203,E501,W503
+ignore = E203,E501,W503,E704
```

Comment:
Without this change, there were rampant complaints about ` E704 multiple statements on one line (def)`

From reading I did, this was a common complaint about black 24 and mixing multiple linters/formatters together that don't agree at the moment. Recommending disabling unless there is a better path forward